go-runtime/compile/build-template/.ftl.tmpl/go/main/main.go.tmpl

```diff
@@ -1,57 +1,47 @@
+{{- $verbs := .Verbs -}}
+{{- $name := .Name -}}
+{{- with .MainCtx -}}
+
```

Comment:
nit: does this newline result in the generated code having an extra newline before the package comment? I'm guessing it's fine since `types.ftl.go.tmpl` has it too, but just wanted to make sure

Reply:
just separated to make the template more readable; the dash at the end of `{{- with .MainCtx -}}`  trims any whitespace/newlines after the block, so it doesn't affect the generated code